### PR TITLE
Amber 22.0 for foss-2022b

### PR DESCRIPTION
**Pull request by DirectorSloan**
Manually git pushed

### CODE REVIEW

File: easybuild/easyconfigs/PnetCDF-1.12.3-gompi-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
```
Comment (branfosj) [inline easybuild/easyconfigs/PnetCDF-1.12.3-gompi-2022b.eb:1]: Please move this into the correct directory `easybuild/easyconfigs/p/PnetCDF/PnetCDF-1.12.3-gompi-2022b.eb`

File: easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2022b-AmberTools-22.3.eb [hunk 2]
```diff
@@ -0,0 +1,96 @@
+name = 'Amber'
+local_amber_ver = 22
+local_ambertools_ver = 22
+# Patch levels from https://ambermd.org/AmberPatches.php and https://ambermd.org/ATPatches.php
+patchlevels = (3, 0)  # (AmberTools, Amber)
```
Comment (branfosj) [inline easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2022b-AmberTools-22.3.eb:5]: Please update this to include the latest patches, so `patchlevels = (5, 3)` ([AmberTools](https://ambermd.org/ATPatches.php), [Amber](https://ambermd.org/AmberPatches.php))

This means that the filename should be altered accordingly, to be `Amber-22.3-foss-2022b-AmberTools-22.5.eb`

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2022b-AmberTools-22.3.eb (hunk 2 above)
Comment (DirectorSloan) [reply easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2022b-AmberTools-22.3.eb:5]: This software tooks me days to run on our cluster.
Happy to got it work with 2022b stuff. Can't test it with AmberTools 22.5 now either can change the productive work environment for our chemican staff now.

Maybe we just drop the PR or AmberTools 22.5 variant may be tested elsewhere.

File: easybuild/easyconfigs/PnetCDF-1.12.3-gompi-2022b.eb (hunk 1 above)
Comment (DirectorSloan) [reply easybuild/easyconfigs/PnetCDF-1.12.3-gompi-2022b.eb:1]: Oh dear, stupid, mistake, will change this. Thanks

**Comment by DirectorSloan** [comment]
Don't bring that upstream in favor to https://github.com/easybuilders/easybuild-easyconfigs/pull/18410

**Pull request closed without merging**